picard/ui/options/cover.py

```diff
@@ -141,7 +59,31 @@ def __init__(self, parent=None):
         self.ui = Ui_CoverOptionsPage()
         self.ui.setupUi(self)
         self.ui.save_images_to_files.clicked.connect(self.update_filename)
-        self.ui.restrict_images_types.clicked.connect(self.update_caa_types)
+
+    def updateProvidersOption(self, items):
+        new = []
+        for item in items:
+            new.append((item.data, item.checked))
+        config.setting['ca_providers'] = new
+
+    def load_ca_providers(self):
+        """
+            Load available providers, initialize tabs, restore state of each
```

Comment:
From a quick grep, it looks like the comment style for methods on classes is to put the content immediately after `"""`.